src/hope_dedup_engine/apps/api/admin/deduplicationset.py

```diff
@@ -45,11 +46,12 @@ class DeduplicationSetAdmin(AdminFiltersMixin, ExtraButtonsMixin, ModelAdmin):
     def has_add_permission(self, request):
         return False
 
-    @button(label="Process")
+    @button(permission=is_root)
```

Comment:
@vitali-yanushchyk-valor  let's do have a function
if the status is failed you can rerun if superuser
if you are root you can run in any status